libs/expo/betterangels/src/lib/ui-components/RequestedProvidedServices/ServiceCheckbox.tsx

```diff
@@ -1,50 +1,45 @@
 import { Checkbox, TextRegular } from '@monorepo/expo/shared/ui-components';
 import { StyleSheet, View } from 'react-native';
-import { ServiceEnum } from '../../apollo';
-import { enumDisplayServices } from '../../static';
+
+type TItem = { id: string; label: string };
+
+type SelectedService = {
+  requestId?: string;
```

Comment:
please rename to `serviceRequestId`. Abbreviating these names can lead to confusion and problems down the road.